addons/l10n_latam_invoice_document/models/account_journal.py

```diff
@@ -28,7 +28,7 @@ def _onchange_company(self):
     @api.constrains('l10n_latam_use_documents')
     def check_use_document(self):
         for rec in self:
```

Comment:
Don't need to loop
```python
def check_use_document(self):
    if self.env['account.move'].search([('journal_id', '=', self.ids), ('state', '!=', 'draft')], limit=1):
         raise ValidationError(_(
                    'You can not modify the field "Use Documents?" if there are validated invoices in this journal!'))	                    'You can not modify the field "Use Documents?" if there are validated invoices in this journal!'))
```

Reply:
hmm, you are right, but better "in" maybe? @jjscarafia

Reply:
yes, seams better to me. On most cases the change of this field is going to be made on only one journal, so, not really a big improvement on performance but better coding of course.